src/client/js/Dialogs/Import/ImportDialog.js

```diff
@@ -150,6 +150,7 @@ define(['js/Loader/LoaderCircles',
 
                     var checkResult = WebGMEGlobal.Client.checkImport(parsedJSONFileContent,
                         CORE_CONSTANTS.EXPORT_TYPE_LIBRARY);
+                    checkResult = null;
```

Comment:
Yes, or at least it needs serious refactoring.
On the other hand, this whole file became deprecated, so I believe this change is just some early residue...